proto/account/tx.proto

```diff
@@ -8,6 +8,18 @@ option go_package = "github.com/tendermint/spn/x/account/types";
 // Msg defines the Msg service.
 service Msg {
     // this line is used by starport scaffolding # proto/tx/rpc
+  rpc CreateCoordinator(MsgCreateCoordinator) returns (MsgCreateCoordinatorResponse);
 }
 
-// this line is used by starport scaffolding # proto/tx/message
\ No newline at end of file
+// this line is used by starport scaffolding # proto/tx/message
+message MsgCreateCoordinator {
+  string creator = 1;
+  string address = 2;
+  string identity = 3;
```

Comment:
We can use a single `CoordinatorDescription` field. Having nested type is not a problem for the message.

Although we can't provide this information in the CLI so we must modify the type manually afterward